### PR TITLE
Update game

### PR DESCRIPTION
**Pull request by AmazingMrBill**
pull request

### CODE REVIEW

**Review by AmazingMrBill — COMMENTED** [review]
The changes are nice

**Review by AmazingMrBill — COMMENTED** [review]
Nice app

**Review by AmazingMrBill — COMMENTED** [review]
**###Wow**
Amazing App
_Why is it not reviewing?_

**Comment by AmazingMrBill** [comment]
dang

**Review by Bill-Kipkemboi — COMMENTED** [review]
Nice app

File: index.html
Comment (AmazingMrBill) [inline index.html:1]: okk

File: .github/steps/-step.txt
Comment (AmazingMrBill) [inline .github/steps/-step.txt:1]: nice

**Review by AmazingMrBill — COMMENTED** [review]
its ok

File: README.md
Comment (AmazingMrBill) [inline README.md:1]: OK

File: .github/steps/-step.txt
Comment (AmazingMrBill) [inline .github/steps/-step.txt:1]: it nice

File: index.html
Comment (AmazingMrBill) [inline index.html:1]: this is also nice

**Review by AmazingMrBill — COMMENTED** [review]
done

File: .github/steps/-step.txt
Comment (AmazingMrBill) [reply .github/steps/-step.txt:1]: ok

File: README.md
Comment (AmazingMrBill) [inline README.md:1]: ok

File: index.html
Comment (AmazingMrBill) [inline index.html:1]: ok

Comment (AmazingMrBill) [reply index.html:1]: ok

**Review by AmazingMrBill — COMMENTED** [review]
Final

**Pull request merged**